compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -49,32 +49,6 @@ static llvm::cl::opt<std::string> clTraceDispatch(
                    "occurrences of the dispatch symbol."),
     llvm::cl::init(""));
 
-static llvm::cl::opt<bool> clDemoteI64ToI32(
```

Comment:
I think these are still better suited in Flow. These are IREE restrictions, and dont really fit with what I think the global optimization pipeline does (IREE agnostic global program optimizations). I'd just leave these here and run them before any of the flow passes.